coda-prototype/UIUtils.js

```diff
@@ -491,7 +490,55 @@ var UIUtils = (function() {
         isScrolledToTop: function(tableContainer) {
             return UIUtils.checkVisible($(".message-row").first()[0]);
             //return $("#message-panel").scrollTop() === 0;
-        }
+        },
+
+        /**
+         * Displays the given message in a green banner at the top of the screen.
+         * Success banners are automatically dismissed after a few seconds, and the contents automatically cleared.
+         * @param message HTML to display in the banner.
+         */
+        displayAlertAsSuccess(message) {
+            // Display the alert.
+            let successAlert = $("#alert");
+            successAlert.removeClass("alert-danger").addClass("alert-success");
+
+            let alertContent = $("#alert-content");
+            alertContent.html(message); // Replace the existing message. TODO: queue alerts?
+
+            successAlert.show();
+            $(".tableFloatingHeaderOriginal").hide();
+
+            // TODO: document
```

Comment:
I don't think this needs any documentation? It seems pretty straight forwards

Reply:
The animation is straightforward but the way it is expressed is extremely weird.

I will rewrite to be clearer, and remove this comment.